examples/fluids/README.md

```diff
@@ -140,6 +140,22 @@ The following options are common among all problem types:
   - Use freestream boundary conditions on this list of faces
   -
 
+* - `-stats_enable`
+  - Enable collection of spanwise flow statistics
+  - `false`
+
+* - `-stats_test`
+  - Use manufactured solution for statistics collection (used for testing)
+  - `false`
+
+* - `-stats_collect_interval`
+  - Number of timesteps between statistics collection
+  - `1`
+
+* - `-stats_write_interval`
+  - Number of timesteps between statistics file writing (`-1` means only at end of run)
+  - `-1`
+
```

Comment:
What do you think of an interface like
```yaml
ts_monitor_turbulence_zspan: cgns:stats.cgns
ts_monitor_turbulence_zspan_collect_interval: 3
ts_monitor_turbulence_zspan_write_interval: 300
```

Reply:
Or just `ts_monitor_turbulence` since the "zspan" is inferred from the isoperiodic SF. But I like the "monitor" part because it's a thing that is monitoring the transient solution, not just viewing at the end.

Reply:
I've been a bit inconsistent with the flag/prefix naming between `spanstats` and `stats`. I think it makes sense to still reference spanwise since we may want to keep track of the global domain statistics (or at least the mean of the global domain solution). 

`ts_monitor` sounds fine to me. It'll pair well with the `ts_monitor_solution`. 

I'm not sure about `_turbulence` though. It seems both equivalent to "statistics" while being more constraining. We're not necessarily constrained to turbulence problems with the setup. In the distant future, if we implement a RANS model into the code, we still might want to leverage the statistics gathering, but it won't necessarily be turbulence statistics.

So how about:
```yaml
ts_monitor_spanstats: cgns:stats.cgns
ts_monitor_spanstats_collect_interval: 3
ts_monitor_spanstats_write_interval: 300
```

Reply:
I guess we could also merge `stats_enable` into `ts_monitor_spanstats`.

Reply:
Yeah, I don't think `stats_enable` should be needed. It isn't a separate option for `ts_monitor_solution`, for example.

Your suggestion is fine with me. I suggested `turbulence` because this may be different than stats about entrained particles in a low-Re flow, or about fabric in a granular flow. These option names can evolve, but doing so is a pain for users and causes more errors updating old runs. It's good to look ahead for names that are unambiguous rather than carrying implicit assumptions about the flow regime and thus context of super-generic terms like "stats".

Reply:
Gotcha. I was thinking in a single-phase continuum flow mindset. Yeah, I think `turbulence` makes sense in that respect.

So `ts_monitor_turbulence_spanstats_*` then?

Reply:
Sounds good. That's very clear.

Reply:
Done now.